src/plugins/path/path.c

```diff
@@ -9,9 +9,36 @@
 #include "path.h"
 
 #ifndef HAVE_KDBCONFIG
+
 #include "kdbconfig.h"
+
+static int createModeBits (const char * modes);
+
+static int handleNoUserCase (Key * parentKey, const char * validPath, const char * modes, int modeMask);
+
+static int switchUser (Key * key, Key * parentKey, const char * name, const struct passwd * p);
+
+static int switchGroup (Key * key, Key * parentKey, const char * name, const struct group * gr);
+
 #endif
```

Comment:
`#endif` should be before the function declarations

Reply:
Thank you, CLion method refactor seemed to be not that intelligent